### PR TITLE
Feature/pfg5 57 validation property

### PR DESCRIPTION
**Pull request by FabioG21**
Added changes: 
- Name, Description and Image is required, and not blank.
- The name restrictions are: The name must be between 3 and 32 characters.
- The description restrictions are: The description must be between 15 and 256 characters.

image_2024-10-28_213514917

image_2024-10-28_213631549

### CODE REVIEW

File: src/main/java/com/capitravel/Capitravel/service/impl/PropertyServiceImpl.java [hunk 1]
```diff
@@ -21,55 +24,57 @@ public PropertyServiceImpl(PropertyRepository propertyRepository){
 
     @Override
     public Property getPropertyById(Long id) {
-        Property idProperty = propertyRepository.findById(id).orElse(null);
+        Property property = propertyRepository.findById(id).orElse(null);
 
-        if(idProperty!=null){
-            return idProperty;
+        if(property!=null){
```
Comment (strikerout) [inline src/main/java/com/capitravel/Capitravel/service/impl/PropertyServiceImpl.java]: Respetar los espacios: property != null

File: src/main/java/com/capitravel/Capitravel/model/Property.java [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-import jakarta.persistence.Id;
+import jakarta.persistence.*;
+import lombok.Data;
 import lombok.Getter;
 import lombok.Setter;
 @Entity
 @Getter
 @Setter
-
+@Data
```
Comment (strikerout) [inline src/main/java/com/capitravel/Capitravel/model/Property.java:7]: Si ya tiene el @ Data, podrías sacar los getter y setter no?

**Review by strikerout — CHANGES_REQUESTED** [review]

File: src/main/java/com/capitravel/Capitravel/service/impl/PropertyServiceImpl.java (hunk 1 above)
Comment (FabioG21) [reply src/main/java/com/capitravel/Capitravel/service/impl/PropertyServiceImpl.java]: Hecho. Se agregaron los espacios.

File: src/main/java/com/capitravel/Capitravel/model/Property.java (hunk 2 above)
Comment (FabioG21) [reply src/main/java/com/capitravel/Capitravel/model/Property.java:7]: No tenia claro totalmente la función del @ Data, ya quedó arreglado ese tema.

**Comment by FabioG21** [comment]
Fixed the requested changes, and fix the update metod in property service, this is a method I had not tried before, luckily I realized it in time.  

evidence:

image_2024-10-28_235556505

Update with an id that does not exist

image_2024-10-28_235644337



Evidence for delete method:

Delete id 3:
image_2024-10-29_000001411

Get ID 3 after deleting it:

image

**Review by strikerout — APPROVED** [review]

**Pull request merged**